frontends/precinct-scanner/src/app_root.tsx

```diff
@@ -570,11 +570,22 @@ export function AppRoot({
 
   if (!isPollsOpen) {
     return (
-      <PollsClosedScreen
-        isLiveMode={!isTestMode}
-        showNoChargerWarning={!computer.batteryIsCharging}
-        scannedBallotCount={scannerStatus.ballotsCounted}
-      />
+      <AppContext.Provider
+        value={{
+          electionDefinition,
+          currentPrecinctId,
+          currentMarkThresholds,
+          machineConfig,
+          auth,
+          isSoundMuted,
+        }}
+      >
+        <PollsClosedScreen
+          isLiveMode={!isTestMode}
+          showNoChargerWarning={!computer.batteryIsCharging}
+          scannedBallotCount={scannerStatus.ballotsCounted}
+        />
+      </AppContext.Provider>
```

Comment:
Maybe move this down into the `voterScreen` variable so it gets the context automatically?

Reply:
That would be confusing to me personally, since it's not a voter screen.

I think it would make sense to wrap in this component in `AppContext`, as this is now the fourth time in the file we've had to explicitly do it. I don't really want do that refactor for this PR, however, because I think the current approach is fine and maybe more readable. Although I could write a helper to wrap in `AppContext` if you think that would be helpful.

Reply:
I think it's fine for now